tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
```

Comment:
the reason i say debug is we have had situations where the debug response was useful to find problems with the CLI where the HTTP response is correct but the output of the CLI that's formatted for consumption isn't (due to scema mismatches).

I'm still puzzled by this change - why limit debug output at all.
Should there be an option to dump the output to a file at least?